test/integration/015_cli_invocation_tests/test_cli_invocation.py

```diff
@@ -173,3 +173,104 @@ def _run_simple_dbt_commands(self, project_dir):
         # In case of 'dbt clean' also test that the clean-targets directories were deleted.
         for target in self.config.clean_targets:
             assert not os.path.isdir(target)
+
+
+class TestCLIInvocationWithProfilesAndProjectDir(ModelCopyingIntegrationTest):
+
+    def setUp(self):
+        super().setUp()
```

Comment:
@jtcohen6 :  I need some help with guiding me through the code base.

I tried moving `dbt-profile` to another location, but apparently the `setUp` here expects it to be present. There is a lot happening in that `setUp`, could you point me where to look?